patches/api/0331-Add-more-advancement-API.patch

```diff
@@ -189,6 +199,18 @@ index 17527c2f7bd5b8a5528388a53f2472bc1869c7f3..792d5ab90e9252b4f660434f72f004fa
 +    io.papermc.paper.advancement.AdvancementDisplay getDisplay();
 +
 +    /**
++     * Gets the formatted display name for this display. This
```

Comment:
jd should include that when a display is present it will return the same as AD#displayName